### ISSUE TITLE
Post inflection bug communication and retry strategy

### ISSUE THREAD

**Issue by luke-gcio** [department-of-veterans-affairs/va.gov-team#18510]
## Problem Statement
Try to get approval to possibly retry and at least communicate to veterans impacted by [this bug](https://github.com/department-of-veterans-affairs/va.gov-team/issues/17595).

Affected Groups
1. Current in progress forms that have not submitted (not tried to submit).
2. Veterans that have tried to submit and have failed.

## Tasks
1. Get list of affected users
2. Cross reference users against other submissions
3. Work with stakeholder on approval for messaging and retrying, separately. 
4. Provide guidance on plan - example email and retry method (think of potential risks of doing this as well)

## Acceptance Criteria
- [ ] Approval for plan from stakeholders

**Comment by annaswims** [comment]
I have a CSV with 6,782 veterans that may have been affected by the bug. It has their in-progress form id, their user uuid, and email address.

**Comment by annaswims** [comment]
For more historical context: there was a platform bug where user data was slightly corrupted.  "Hello there, Sam-I-Am!"  would become "hello there, samIAm!".  If a user tried to submit the form it might not have been successful. We fixed the bug and the data on January 14 so users who were not able to submit in the past should be able to click submit now and have a successful submission. It'd be nice to tell users that we found and fixed a problem affecting their ability to submit.
